Resources/ServerInfo/Guidebook/Mobs/Diona.xml

```diff
@@ -0,0 +1,25 @@
+<Document>
+  # Diona
+
+  <Box>
+    <GuideEntityEmbed Entity="MobDiona" Caption=""/>
+  </Box>
+
+  They can't wear shoes, but are not slowed by Kudzu.
+  They get hungry and thirsty slower.
+  Their "blood" is normal water.
+  Weed killer poisons them, while Robust Harvest heals them.
+
+  They take [color=#1e90ff]30% less Blunt damage and 20% less Slash damage[/color]; but [color=#ffa500]50% more Heat damage, 20% more Shock damage, and they easily catch on fire.[/color]
```

Comment:
I'd explicitly say that high heat damage can ignite them as that's fairly unique.